docs/_config.yml

```diff
@@ -306,6 +305,7 @@ es:
         - path: es/week12/12-2.md
         - path: es/week12/12-3.md
     - path: es/week13/13.md
+      sections:
```

Comment:
This has been already fixed https://github.com/Atcold/pytorch-Deep-Learning/commit/b3e890439d7ad9402513c41ed4ceef2c620e208e